### PR TITLE
Remove redundant user activation text from PointerLock spec.

### PR DESCRIPTION
**Pull request by mustaqahmed**
Closes #70 

Web platform tests not needed: HTML spec already covers user activation triggering event.

Implementation commitment not applicable: this is a backward compatible change.

***
<a href="https://pr-preview.s3.amazonaws.com/mustaqahmed/pointerlock/pull/76.html" title="Last updated on Nov 15, 2021, 8:18 PM UTC (438a679)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerlock/76/b47d8dd...mustaqahmed:438a679.html" title="Last updated on Nov 15, 2021, 8:18 PM UTC (438a679)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -213,49 +191,45 @@ <h2>
...
-            <p data-link-for="Element">
-              Pointer lock is commonly combined with fullscreen [[FULLSCREEN]],
-              and if an <a>engagement gesture</a> is used to enter fullscreen
-              it is sufficient for a subsequent {{requestPointerLock()}}.
-            </p>
+          <aside class="note">
+            When a single <a>user activation</a> initiates both pointer lock and
+            fullscreen [[FULLSCREEN]], the {{requestPointerLock()}} call
+            succeeds only when it is made before a fullscreen request because
+            fullscreen is a <a>transient activation-consuming api</a>.
```
Comment (domenic) [inline index.html]:
```suggestion
            fullscreen is a <a>transient activation-consuming API</a>.
```

File: index.html [hunk 2]
```diff
@@ -213,49 +191,45 @@ <h2>
...
           </p>
           <p>
-            If a user has exited pointer lock via the <a>default unlock
-            gesture</a>, or pointer lock has not previously been entered for
-            this document, an event generated as a result of an <a>engagement
-            gesture</a> must be received by the document before
-            {{requestPointerLock()}} will succeed.
+            Pointer lock is a <a>transient activation-gated API</a>, therefore a
+            {{requestPointerLock()}} call MUST fail if <a>transient
+            activation</a> is not available.  This prevents locking upon initial
```
Comment (domenic) [inline index.html]: Maybe be more specific, i.e. say that it must fail if this's relevant global object does not have transient activation?

**Review by domenic — APPROVED** [review]
LGTM with nits. An algorithm with steps would of course be nicer, but this is a definite improvement :)

File: index.html (hunk 2 above)
Comment (mustaqahmed) [reply index.html]: Done.

**Pull request merged**